python/ray/actor.py

```diff
@@ -235,9 +235,10 @@ def _bind(
         num_returns=None,
         concurrency_group=None,
         _generator_backpressure_num_objects=None,
-    ):
+    ) -> Union["ray.dag.ClassMethodNode", Tuple["ray.dag.ClassMethodNode", ...]]:
```

Comment:
(1) seems better to me because it is less error-prone and it follows the normal non-DAG Ray API better. InputNode is kind of a special case because there's no "root" task (and actually we've already seen user errors there when not all indices are used).